include/blazesym.h

```diff
@@ -94,6 +94,15 @@ typedef enum blazesym_src_type {
  */
 typedef struct blazesym blazesym;
 
+/**
+ * An inspector of various "sources".
+ *
+ * Object of this type can be used to perform inspections of supported sources.
+ * E.g., using an ELF file as a source, information about a symbol can be
+ * inquired based on its name.
+ */
+typedef struct Inspector Inspector;
```

Comment:
so we have "blazesym" for symbolizer, right? Should this be all lower case and with "blaze" prefix as well? Just plain "Inspector" is too generic and in C code base might cause too many naming conflicts.

btw, what did you plan to call Normalizer for C API? Let's make all of them named in a consistent manner.

Reply:
Should be fixed now. Normalizer is called `blaze_normalizer`.